code-snippet-plugin/src/main/java/com/okta/maven/snippet/CodeSnippetMojo.java

```diff
@@ -115,15 +118,30 @@ public void execute() throws MojoExecutionException {
 
         final Map<String, String> methodBodyMap = new HashMap<>();
 
-
         new VoidVisitorAdapter<Object>() {
             @Override
             public void visit(final MethodDeclaration method, final Object arg) {
                 super.visit(method, arg);
 
                 String body = method.getBody().get().getStatements().stream()
-                        .map(Node::toString)
+                        .map(statement -> {
+                            TokenRange tokenRange = statement.getTokenRange().get();
+                            Optional<Comment> comment = statement.getComment();
+
+                            StringBuilder blockAString = new StringBuilder();
+                            if ( comment.isPresent()) {
```

Comment:
`if ( comment.isPresent())` -> `if (comment.isPresent())`
^ Remove the extra space